src/osd/OSDMap.h

```diff
@@ -275,12 +275,10 @@ class OSDMap {
   }
 
   // no copying
-  /* oh, how i long for c++11...
 private:
   OSDMap(const OSDMap& other) = default;
```

Comment:
if everything builds with = delete, let's do that.  I forget if there is a private user.